fxa-auth-db-server/test/backend/remote.js

```diff
@@ -771,18 +785,6 @@ module.exports = function(cfg, server) {
         })
         .then(function (r) {
           t.equal(r.obj.tokenVerificationId, null, 'tokenVerificationId is null')
-
-          // Attempt to verify the verified key fetch token
-          return client.postThen('/tokens/' + verifiedUser.keyFetchToken.tokenVerificationId + '/verify', {
```

Comment:
This was calling `/tokens/undefined/verify` which is a rather pointless thing to test :-)